k8s/service_controller/model/service_instance.go

```diff
@@ -22,9 +22,9 @@ type LastOperation struct {
 }
 
 type CreateServiceInstanceRequest struct {
```

Comment:
Doug, this is broken in a bunch of places and I think we're going to follow up with a full model cleanup, which separates sc API model from sb API model.